microsoft/testsuites/vm_extensions/linux_patch_extension.py

```diff
@@ -183,6 +203,10 @@ def verify_vm_assess_patches(
         # service
         _verify_vm_agent_running(node, log)
 
+        log.debug(
+            f"what is node.os.information.major: {node.os.information.version.major}"
+        )
```

Comment:
is it needed? you can remove it if it is for debug purpose.

Reply:
removed